gaphor/diagram/presentation.py

```diff
@@ -75,6 +75,9 @@ def revert(self, target):
 
 
 class HandlePositionUpdate:
+
+    _lastHandleMoved = None
```

Comment:
Can you change this to underscore notation (snake case):

```suggestion
    _last_handle_moved = None
```